src/ansys/aedt/core/workflows/hfss/choke_designer.py

```diff
@@ -368,6 +368,14 @@ def main(extension_args):
     # Create choke geometry
     list_object = hfss.modeler.create_choke(str(json_path))
 
+    if not list_object:  # pragma: no cover
+        if temp_dir.exists():
```

Comment:
```suggestion
        app.logger.error("No object associated to chocke creation.")
        if temp_dir.exists():
```
Would it make sense to add logging in that case scenario since we are stopping the workflow ?